antlr-kotlin-runtime/build.gradle.kts

```diff
@@ -0,0 +1,78 @@
+plugins {
+    id("org.jetbrains.kotlin.multiplatform")
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+group = "org.antlr"
+version = "0.0.1"
+
+apply(plugin = "maven-publish")
+
+kotlin {
+    jvm()
+    js {
+        browser {
+        }
+        nodejs {
+        }
+    }
+    // For ARM, should be changed to iosArm32 or iosArm64
```

Comment:
We should add this explanation also in the Readme, I think

Reply:
That comment autogenerates with every multiplatform lib. Currently I haven't done the native target, so it's not yet reasoned